vzlogger_buildroot_builder.Dockerfile

```diff
@@ -0,0 +1,244 @@
+# Dockerfile to build a buildroot-based cross-compile environment for vzlogger
+
+# the OS version used here is largely irrelevant,
+# as buildroot compiles it's own toolchain
+ARG	DEBIAN_VERSION=buster
+FROM	buildpack-deps:$DEBIAN_VERSION as builder
+
+LABEL	maintainer="Thorben T. <r00t@constancy.org>"
+
+# don't inherit upstream labels
+LABEL	org.label-schema.build-date=null
+LABEL	org.label-schema.license=null
+LABEL	org.label-schema.name=null
+LABEL	org.label-schema.schema-version=null
+LABEL	org.label-schema.vendor=null
+
+# install host-OS packages required by buildroot
+# (buildpack-deps already contains all but bc, cpio and rsync)
+RUN	set -xe ; \
+	sed -i '/^deb/s/ / [trusted=yes] /' /etc/apt/sources.list ; \
+	apt-get update ; \
+	apt-get -y --force-yes install \
+		# https://buildroot.org/downloads/manual/manual.html#requirement
+		#make gcc g++ \
+		#patch \
+		#bzip2 \
+		cpio \
+		#unzip \
+		rsync \
+		#file \
+		bc \
+		#libncurses-dev \
+		#wget \
+		#git \
+		# buildroot will build it's own cmake because it's newer than debian's
+		##cmake \
+	; \
+	apt-get purge ; \
+	rm -fr /var/lib/apt/lists
+
+WORKDIR	/buildroot
+
+RUN \
+	set -xe ; \
+	\
+	# download and unpack buildroot
+	wget --progress=dot:mega https://buildroot.org/downloads/buildroot-2021.02.1.tar.bz2 ; \
+	tar xj --strip-components=1 <buildroot-2021.02.1.tar.bz2 ; \
+	rm buildroot-2021.02.1.tar.bz2
+
+# install a helper for neatly formatted build output
+RUN \
+	set -xe ; \
+	( \
+	echo '#!/bin/bash' ;\
+	echo 'f=/tmp/br_make_wrapper.$$' ; \
+	echo 'trap "rm -f $f" EXIT' ; \
+	echo 'echo "=== make $@ ===" >&2' ; \
+	echo 'make "$@" &> >(tee "$f" | grep --line-buffered ">>>") || {' ; \
+        echo '    e=$?' ; \
+        echo '    cat "$f" >&2' ; \
+        echo '    exit $e' ; \
+	echo '}' ; \
```

Comment:
Would easier to maintain if this file would be a normal file and copied with `ADD` into the image?

Reply:
i authored this particular script in 2020, and it never needed an a change since.

other than that, i inlined all the files in the dockerfile to avoid cluttering the repo with single-purpose files.
but i guess we should create a subdirectory for this anyway, and could then split them out.